uitestutils/src/main/java/com/gravatar/uitestutils/RoborazziTest.kt

```diff
@@ -62,9 +61,10 @@ abstract class RoborazziTest {
     }
 
     fun screenshotTest(composable: @Composable () -> Unit) {
-        captureRoboImage {
+        composeRule.setContent {
```

Comment:
I wasn't able to continue using the fun `captureRoboImage` function. Look like the library does something different 🤷‍♂️